nakama/main.go

```diff
@@ -115,6 +121,38 @@ func initReceiptDispatcher(log runtime.Logger) error {
 	return nil
 }
 
+func initEventHub(ctx context.Context, log runtime.Logger, nk runtime.NakamaModule) error {
+	globalEventHub, err := createEventHub()
```

Comment:
The := here means it's shadowing the globalEventHub variable in main.go.

Reply:
Yeah you're right. Bug since event hub is not used anywhere else and the tests still passed I'm going to delete the global and make this local.